src/klein/_decorators.py

```diff
@@ -19,11 +24,15 @@ def bindable(bindable):
     @return: its argument, modified to mark it as unconditinally requiring an
         instance argument.
     """
-    bindable.__klein_bound__ = True
+    bindable.__klein_bound__ = True  # type: ignore[attr-defined]
     return bindable
 
 
-def modified(modification, original, modifier=None):
+def modified(
+    modification, original, modifier=None
+):
+    # FIXME: This maybe isn't quite right
+    # __type: (Text, C, Optional[Callable[[C], C]]) -> Callable[[C], C]
```

Comment:
@glyph maybe you wanna take a stab at this one.  I did some flailing and gave up.

Reply:
It doesn't need to happen in this PR though; we can always add it later.

Reply:
The docstring states that the returned callable doesn't need to have the same signature as the original. In that case, `original` should probably be annotated as having type `Callable` instead of `C`.

Reply:
Aha, thanks.